tests/tests_insights_client_register.yml

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Basic insights-client register/unregister
+  hosts: all
+  gather_facts: true
+  become: true
+  tags:
+    - tests::slow
+  tasks:
+    - name: Unregister insights
```

Comment:
Is there a test to register a system with insights?

Reply:
I removed the test to register insights-client here because it is implicit  in the [test_register_unregister](https://github.com/linux-system-roles/rhc/blob/main/tests/tests_register_unregister.yml)